### PR TITLE
fixed lint issues for lower CRA version and remove warning on browser…

### PR DESCRIPTION
**Pull request by ztcrypto**
… console too.

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -8,12 +8,7 @@ import gtmConfig from './utils/googleTagManager'
 
 TagManager.initialize(gtmConfig)
 
-ReactDOM.render(
-  <React.StrictMode>
```
Comment (controtie) [inline src/index.tsx:12]: what's the intended effect of removing strict mode?

File: src/views/CGI/CGI.tsx [hunk 2]
```diff
@@ -143,15 +143,15 @@ const CgiProductPage = (props: { title: string }) => {
...
+            <div>
               The following rounding of data are used for the Index calculation
               and publication:
               <ul>
                 <li>Prices are rounded to eight significant figures</li>
                 <li>Weights are rounded to four decimal places</li>
                 <li>Index levels are rounded to two decimal places</li>
               </ul>
-            </p>
+            </div>
```
Comment (controtie) [inline src/views/CGI/CGI.tsx:154]: does this change how the page is rendered?

Comment (controtie) [reply src/views/CGI/CGI.tsx:154]: i.e. is the spacing still good/does it need to be fixed with further styling

**Review by controtie — COMMENTED** [review]
looks mostly good. left some minor comments.
Also, why is yarn.lock updated but no packages in package.json have changed?

File: src/views/CGI/CGI.tsx (hunk 2 above)
Comment (ztcrypto) [reply src/views/CGI/CGI.tsx:154]: it's good yeah

File: src/index.tsx (hunk 1 above)
Comment (ztcrypto) [reply src/index.tsx:12]: just to remove browser console issue of unsafe_componentunmount warning.
actually it's ok to remain as it or remove.

**Review by controtie — APPROVED** [review]
LGTM

**Pull request merged**